Sua Musica/dist/metadata.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": {
+        "name": "milrak",
+        "id": "401529002403037185"
+    },
+    "service": "Sua Música",
+    "description": {
+        "en": "Download the best launching platform for independent artists in Brazil for free. Concerts, Promotional CDs, Music, Video Clips, Instagram and news from Forró, Sertanejo, Axé and others. Create a free Profile for your Band right now.",
+        "pt_BR": "Baixe grátis na melhor plataforma de lançamento de Artistas independentes do Brasil. Shows, CDs Promocionais, Músicas, Vídeo Clipes, Instagram e novidades do Forró, Sertanejo, Axé e outros. Crie um Pefil grátis para sua Banda agora mesmo."
+    },
+    "url": "suamusica.com.br",
+    "version": "1.0.0",
+    "logo": "https://i.imgur.com/MkYz91m.png",
+    "thumbnail": "https://i.imgur.com/ivNJONz.png",
```

Comment:
can you change the thumbnail to something related to the website?
(e.g: a screenshot or some promotional image)